server/VERSION

```diff
@@ -1,2 +1 @@
-0.69.2
-
+0.69.4
```

Comment:
Possibly doesn't matter that there's no newline at the end; but GitHub is complaining, and some other utilities might as well.

Reply:
Fixed.